components/base/__tests__/FilterTabs.test.tsx

```diff
@@ -1,18 +1,18 @@
 import renderer from "react-test-renderer";
-import ReactTestUtils from "react-dom/test-utils";
-import FilterTabs from "../FilterTabs";
+import FilterTabs, { TabStyle } from "../FilterTabs";
 
-test("Filter Tabs", () => {
+const TABS = ["GET BUILDING", "PREREQUISITES", "FEATURES"];
+
+test("FilterTabs", () => {
```

Comment:
I renamed this to match the component name because some IDEs/editors will find matching string occurrences like this when/if you rename a component and update them for you.